.github/workflows/build.yml

```diff
@@ -40,3 +40,72 @@ jobs:
           types: rpm
           maven-username: ${{ secrets.NEXUS_USERNAME }}
           maven-password: ${{ secrets.NEXUS_PASSWORD }}
+
+  build_centoslinux:
+    name: Centos7
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build
+        id: build
+        uses: ./.github/actions/centos-build
+      - uses: actions/upload-artifact@v3
+        with:
+          name: centos-rpms
+          path: rpms/*.rpm
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: rpms/*.rpm
+      - name: Deploy to nexus
+        uses: Alfresco/alfresco-build-tools/.github/actions/maven-deploy-file@v1.11.0
+        if: startsWith(github.ref_name, 'refs/tags/')
+        with:
+          group-id: org.imagemagick
+          artifact-id: imagemagick-distribution
+          repository-url: https://nexus.alfresco.com/nexus/content/repositories/thirdparty/
+          version: ${{ steps.build.outputs.built-version }}-ci
+          file: rpms/ImageMagick-${{ steps.build.outputs.built-version }}.x86_64.rpm
+          classifier: centos7
+          files: rpms/ImageMagick-libs-${{ steps.build.outputs.built-version }}.x86_64.rpm
+          classifiers: centos7-libs
+          types: rpm
+          maven-username: ${{ secrets.NEXUS_USERNAME }}
+          maven-password: ${{ secrets.NEXUS_PASSWORD }}
+
+  build_Ubuntu:
+    name: Ubuntu-18.04
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build
+        id: build
+        uses: ./.github/actions/Ubuntu-build
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ubuntu-deb
+          path: rpms/*.deb
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: rpms/*.deb
```

Comment:
this one can't work when run on multiple step, solutions can be:
* add a new final job that just to this Release step in which we download the previous job artifacts and then release
* just remove it since it was not really a requirement (we started with that and then moved to nexus)

Reply:
IMHO we should skip using github releases completely. Again here we have a repo with multiples artifacts which can/should be released separately and might not even follow the same version... That's going to be a mess to deal with and we can afford not having a global release here. What do you think?

Reply:
after merging this PR we could create a v7.1.0-16~1 but in the end I think we really need to stick to tags that represent repository versions and not the version of artifacts we are delivering with the repo.

@slohe1 remove all the usages of `action-gh-release` and then after merging I will take care of (re)creating the tag